bodyfetcher.py

```diff
@@ -1,7 +1,7 @@
 # coding=utf-8
```

Comment:
We don't use the `queue_timings` variable on line 23 anymore, right? Can we just get rid of it?